features/step_definitions/generic_steps.rb

```diff
@@ -92,25 +92,47 @@
   step %{I should see "success"}
 end
 
-def assure_xpath_present(tag, attribute, value, selector)
+def xpath_for_tag_with_attributes(tag, attributes)
```

Comment:
If we anticipated frequently needing to check multiple attributes, this would probably be the way to go, but I think we're getting a bit ahead of ourselves here. 

Rather than modify all these existing steps, we can define one new step to check for these meta tags (and a corresponding one to check that the tags aren't present):

```gherkin
Then "the page should be hidden from search engines" do
  page.should have_xpath('//head/meta[@name="robots"][@content="noindex"]', visible: false)
  page.should have_xpath('//head/meta[@name="googlebot"][@content="noindex"]', visible: false)
end
```

Reply:
+1. Additionally, we should prefer [the expect syntax](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers), as well as CSS selectors (more familiar than XPath), so something like:
```rb
expect(page).to have_css("meta[name=robots][content=noindex]", visible: false)
```